node/pkg/guardiansigner/benchmarksigner.go

```diff
@@ -0,0 +1,59 @@
+package guardiansigner
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"time"
+)
+
+type BenchmarkSigner struct {
```

Comment:
Please, document this.

Reply:
IIUC this means we can benchmark AWS KMS signer by passing its configuration to a signer of type `benchmark`, correct?